src/vs/workbench/api/common/extHost.protocol.ts

```diff
@@ -407,7 +407,7 @@ export interface MainThreadLanguageFeaturesShape extends IDisposable {
 	$registerSelectionRangeProvider(handle: number, selector: IDocumentFilterDto[]): void;
 	$registerCallHierarchyProvider(handle: number, selector: IDocumentFilterDto[]): void;
 	$registerTypeHierarchyProvider(handle: number, selector: IDocumentFilterDto[]): void;
-	$registerDocumentOnDropEditProvider(handle: number, selector: IDocumentFilterDto[]): void;
+	$registerDocumentOnDropEditProvider(handle: number, selector: IDocumentFilterDto[], isBuiltin: boolean): void;
```

Comment:
A little strange to only respect this new option for drop providers. If we think this makes sense, perhaps we could extend it other provider where it makes sense too

Reply:
Yeah, I was expecting to go all in and make this an optional property of the `LanguageFilter` type so that this applies to all providers